archiver/biosamples.py

```diff
@@ -14,10 +27,15 @@ def _taxon(*args):
 
 def derive_concrete_type(*args):
     schema_url = args[0]
-    concrete_type = schema_url.split('/')[-1]
+    concrete_type = get_concrete_type(schema_url)
```

Comment:
(May be a noob question because I'm not a developer!) 

what does the addition of this function accomplish? isn't it just doing the same but in more lines?

Reply:
lol you have a point, but i guess if i have to use it again, the benefit of this is i don't have to remember how to split it and if ever the url format changed i need to only change it in one place.

Reply:
That sounds sensible, thanks!